bin/deb_local

```diff
@@ -377,13 +377,13 @@ while pkgs_added
             next
         end
 
-        if ancestor_blacklist_patterns.select do |pat|
+        if !(ancestor_blacklist_patterns.select do |pat|
```

Comment:
Not very readable, better use a local variable to cache the results.

Reply:
Just learned about any?. I would thing replacing !(array.select {} .empty?) by array.any? {} is better?

Reply:
Yes, that should do.